avaframe/ana3AIMEC/dfa2Aimec.py

```diff
@@ -56,15 +56,13 @@ def extractMBInfo(avaDir):
     # Get info from ExpLog
     logName = os.path.join(avaDir, 'Outputs', 'com1DFA', 'ExpLog.txt')
     logDictExp = fU.readLogFile(logName)
-    simName = []
-    for name in logDictExp['simName']:
-        simName.append(name.split('_')[0])
-    relNames = set(simName)
+    names = logDictExp['fullName']
+    simNames = sorted(set(names), key=lambda s: s.split("_")[3])
 
     # Read mass data from log and save to file for each simulation run
     countFile = 0
-    for relName in relNames:
-        log.info('These are the release areas: %s ' % (relName))
+    for simName in simNames:
+        log.info('These are the sim Names areas: %s ' % (simName))
```

Comment:
Either just:
simName:
or 
simulation names:

or 
This is the simulation name:

Reply:
I changed it to 'This is the simulation name...'